### DISCUSSION TITLE
关于通知方式扩展的点子

### DISCUSSION THREAD

**Discussion by grainGods** [marvinSpring/spring-boot-starter-calma#10] [Ideas]
你好，我觉得可以现在很多情况企业或个人用钉钉和短信通知是否太过于昂贵或者繁杂，但是可以通过邮箱的方式通知，可以扩展一个用邮箱方式通知的功能

**Comment by marvinSpring** [comment]
额，实际上calma是支持扩展的，并且你只需要扩展其model和config即可，不过目前扩展性确实有待提高

**Comment by marvinSpring** [comment]
![image](https://user-images.githubusercontent.com/61732476/236596003-ed5cd920-0cb6-4eec-8452-57d511377d08.png)

**Comment by marvinSpring** [comment]
> 你好，我觉得可以现在很多情况企业或者个人用钉钉和短信息通信知道是不是太过于昂贵或者繁杂，但是是可以通过邮箱的方式通信的知道，可以扩展一个使用邮箱方式通知的功能

我已经在讨论区回复了你看下
